src/AspNet.Security.OAuth.GitCode/AspNet.Security.OAuth.GitCode.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+  </PropertyGroup>
+
+  <!-- TODO Remove once published to NuGet.org -->
+  <PropertyGroup>
+    <PackageValidationBaselineVersion>9.1.0</PackageValidationBaselineVersion>
```

Comment:
Moved because we need that element ongoing, rather than it being temporary (which `DisablePackageBaselineValidation` is).

```suggestion
```